src/annotations/AnnotationParser.php

```diff
@@ -122,7 +122,7 @@ public function parse($source, $path)
                         $use .= $str;
                     } elseif ($type === self::CHAR) {
                         if ($str === ',' || $str === ';') {
-                            $uses[substr($use, 1 + strrpos($use, '\\'))] = $use;
+                            $uses[substr($use, 1 + (strrpos($use, '\\') ?: -1))] = $use;
```

Comment:
Maybe it's better to be written as `if/else` without ternary operator to be more readable. Now the `1 + -1` construct that is evaluated at the end is a bit confusing.